addons/account/account_move_line.py

```diff
@@ -89,6 +89,17 @@ def _compute_cash_basis(self):
                 move_line.credit_cash_basis = move_line.credit
             move_line.balance_cash_basis = move_line.debit_cash_basis - move_line.credit_cash_basis
 
+    @api.one
+    @api.depends('move_id.line_id')
+    def _get_counterpart(self):
+        counterpart = set()
+        for line in self.move_id.line_id:
+            if (line.account_id.code != self.account_id.code):
+                counterpart.add(line.account_id.code)
+        if len(counterpart) > 2:
+            counterpart = counterpart[0:2] + ["..."]
```

Comment:
didn't you forget to cast counterpart as a list before doing counterpart[0:2] ??